vdb-bench-assembly/dev-server/json/vdbs.json

```diff
@@ -1,15 +1,15 @@
 {
     "vdbs": [
         {
-            "id": "portfolio",
+            "id": "Portfolio",
             "_links": [
                 {
                     "rel": "self",
-                    "href": "/vdbs/portfolio"
+                    "href": "/vdbs/Portfolio"
                 },
                 {
                     "rel": "content",
-                    "href": "/vdb/portfolio"
+                    "href": "/vdb/Portfolio"
```

Comment:
I am still not understanding why the "content" has to be under a different URI root. If "content" is the XML file, couldn't the URI just be something like "/vdbs/Portfolio/vdb.xml"?